fenix/app/src/main/java/org/mozilla/fenix/tabstray/TabsTray.kt

```diff
@@ -64,12 +76,17 @@ fun TabsTray(
             ) { position ->
                 when (Page.positionToPage(position)) {
                     Page.NormalTabs -> {
-                        Text(
-                            text = "Normal tabs",
-                            modifier = Modifier.padding(all = 16.dp),
-                            color = FirefoxTheme.colors.textPrimary,
-                            style = FirefoxTheme.typography.body1,
-                        )
+                        FirefoxTheme(theme = Theme.getTheme(allowPrivateTheme = false)) {
```

Comment:
❓ Do we need this? The theme is already a top level composable in `TabsTrayFragment` ?

Reply:
This is to manually prevent the private theme from being used on the normal tabs page. I'm currently discussing with UX for another ticket, Bug 1820897, what parts of the Tabs Tray, if any, will actually utilize the private theme.

Reply:
okay! thanks for the explanation!